fedimint-client/Cargo.toml

```diff
@@ -39,7 +39,7 @@ serde_json = { workspace = true }
 strum = { workspace = true }
 strum_macros = { workspace = true }
 thiserror = { workspace = true }
-tokio = { version = "1.36.0", features = [ "time", "macros" ] }
+tokio = { version = "1.36.0", features = [ "time", "macros", "rt" ] }
```

Comment:
tokio has runtime on wasm :eyes: does it work?

Reply:
https://github.com/fedimint/fedimint/pull/4839/files landed so i hope so.

Reply:
lets see https://github.com/fedimint/fedimint/pull/4850